src/dcqc/tests/base_test.py

```diff
@@ -191,7 +129,8 @@ def __post_init__(self, command_args: Sequence[str]):
 
     @property
     def command(self) -> str:
-        return " ".join(self._command_args)
+        args_strings = [str(arg) for arg in self._command_args]
```

Comment:
The only reason I added this is because I changed the tests to include a `Path` in the command arguments, which needs to be converted to a string prior to concatenation.